docs/static/deploying.asciidoc

```diff
@@ -1,146 +1,264 @@
-use a message broker as a buffer. By default, Logstash throttles incoming events when
-indexer consumption rates fall below incoming data rates. Since this throttling can lead to events being buffered at
-the data source, preventing back pressure with message brokers becomes an important part of managing your deployment.
+When running Logstash 5.2 or greater,
+the https://www.elastic.co/products/x-pack/monitoring[Monitoring UI] provides
+deep visibility into your deployment metrics, helping observe performance and
+alleviate bottlenecks as you scale. Monitoring is an X-Pack feature under the
+Basic License and is therefore *free to use*. To get started, consult the
+{xpack}/monitoring-logstash.html[X-Pack Monitoring documentation].
 
-Adding a message broker to your Logstash deployment also provides a level of protection from data loss. When a Logstash
-instance that has consumed data from the message broker fails, the data can be replayed from the message broker to an
-active Logstash instance.
+If external monitoring is preferred, there are <<monitoring,Monitoring APIs>>
+that return point-in-time metrics snapshots.
 
-Several third-party message brokers exist, such as Redis, Kafka, or RabbitMQ. Logstash provides input and output plugins
-to integrate with several of these third-party message brokers. When your Logstash deployment has a message broker
-configured, Logstash functionally exists in two phases: shipping instances, which handles data ingestion and storage in
-the message broker, and indexing instances, which retrieve the data from the message broker, apply any configured
-filtering, and write the filtered data to an Elasticsearch index.
+[float]
+[[adding-other-sources]]
+=== Adding Other Popular Sources
+
+Users may have other mechanisms of collecting logging data, and it’s easy to
+integrate and centralize them into the Elastic Stack. Let’s walk through a few
+scenarios:
 
-image::static/images/deploy_5.png[]
+image::static/images/deploy3.png[]
 
 [float]
-[[deploying-logstash-ha]]
-=== Multiple Connections for Logstash High Availability
+==== TCP, UDP, and HTTP Protocols
 
-To make your Logstash deployment more resilient to individual instance failures, you can set up a load balancer between
-your data source machines and the Logstash cluster. The load balancer handles the individual connections to the
-Logstash instances to ensure continuity of data ingestion and processing even when an individual instance is unavailable.
+The TCP, UDP, and HTTP protocols are common ways to feed data into Logstash.
+Logstash can expose endpoint listeners with the respective
+<<plugins-inputs-tcp,TCP>>, <<plugins-inputs-udp,UDP>>, and
+<<plugins-inputs-http,HTTP>> input plugins. The data sources enumerated below
+are typically ingested through one of these three protocols.
 
-image::static/images/deploy_6.png[]
+//ALVIN: Not sure what we are referring to when we say "the data sources enumerated below"
```

Comment:
network/security data, syslog servers, iot data all come through either the TCP, UDP, or HTTP protocols.